src/services/analytics/useGtm.ts

```diff
@@ -26,6 +26,12 @@ import { DeviceType } from './types'
 import useSafeAddress from '@/hooks/useSafeAddress'
 import useWallet from '@/hooks/wallets/useWallet'
 
+enum GA_USER_PROPERTIES {
```

Comment:
I wouldn’t put it here. This hook is just a hook to bind GTM to React. The actual analytics service is where it should be, similar to our custom event definitions.